st2common/st2common/models/api/execution.py

```diff
@@ -60,7 +60,7 @@ class ActionExecutionAPI(BaseAPI):
             "rule": RuleAPI.schema,
             "action": REQUIRED_ATTR_SCHEMAS["action"],
             "runner": REQUIRED_ATTR_SCHEMAS["runner"],
-            "liveaction": REQUIRED_ATTR_SCHEMAS["liveaction"],
+            "liveaction_id": {"type": "string", "required": True},
```

Comment:
The API does not have to match the database. So, we can leave this as liveaction in the API model but construct it from both execution and liveaction db models. That would make this backwards compatible because the public API would not be changing.

Reply:
I added the embedded liveaction back to the database.